dlnd/dlnd_datamode_rof_forcing_mod.F90

```diff
@@ -18,18 +17,27 @@ module dlnd_datamode_rof_forcing_mod
    public :: dlnd_datamode_rof_forcing_init_pointers
    public :: dlnd_datamode_rof_forcing_advance
 
-   ! module pointer arrays
+   ! export state pointers
    real(r8), pointer :: lfrac(:)
-
-   character(*), parameter :: Flrl_rofsur_nonh2o = 'Flrl_rofsur_nonh2o'
-   character(*), parameter :: Flrl_rofsur        = 'Flrl_rofsur'
-   character(*), parameter :: Flrl_rofsub        = 'Flrl_rofsub'
-   character(*), parameter :: Flrl_rofgwl        = 'Flrl_rofgwl'
-   character(*), parameter :: Flrl_rofi          = 'Flrl_rofi'
-   character(*), parameter :: Flrl_irrig         = 'Flrl_irrig'
-
-   character(len=11) :: fldnames_h2o(5) = &
-        (/'Flrl_rofsur', 'Flrl_rofsub','Flrl_rofgwl','Flrl_rofi  ','Flrl_irrig '/)
+   real(r8), pointer :: Flrl_rofsur_nonh2o_2d(:,:)
```

Comment:
These should all be initialized (` => NULL()`). This also applies to `lfrac` above.